allow user to configure static rundeck tokens file

jardleex opened the pull request:
Hi @mrlesmithjr,

this patch will allow the user to configure a static token file for Rundeck. It will only trigger if the user configures user/key pairs within _rundeck_framework_tokens_.
[framework.properties documentation](http://rundeck.org/docs/administration/configuration-file-reference.html#framework.properties)

Let me know if this is okay for you or if I missed something.

Best regards

Jard

jardleex committed:
allow user to configure static rundeck tokens file

jardleex commented:
Hello @mrlesmithjr 

do you need any further input on this from my side?
I'd really like to get this feature in to use it in our production environment.
Also found new feature to implement too :).

Thank you and best regards

Jard

mrlesmithjr commented:
@jardleex Can you rebase this against master and push back up? I will merge this one too. Thanks again!

jardleex commented:
Hello @mrlesmithjr,

thanks for your feedback. I added the initially missing tokens.properties.j2, squashed the commits and did a rebase. Hopefully I did everything right.
Let me know if I messed something up.

Best regards

Jard

mrlesmithjr commented:
@jardleex Merged. Thanks again man!
